src/App.test.js

```diff
@@ -1,8 +1,11 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { App } from './App';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  render(
+    <Router>
+      <App />
+    </Router>
```

Comment:
Will need to go back and test App. This is where most of the integration testing can take place.